group-coordinator/src/main/resources/common/message/ConsumerGroupPartitionMetadataValue.json

```diff
@@ -29,7 +29,20 @@
       { "name": "TopicName", "versions": "0+", "type": "string",
         "about": "The topic name." },
       { "name": "NumPartitions", "versions": "0+", "type": "int32",
-        "about": "The number of partitions of the topic." }
+        "about": "The number of partitions of the topic." },
+      { "name": "PartitionRacks",
+        "versions": "0+",
+        "type": "[]PartitionMetadata",
+        "about": "Partitions mapped to a set of racks.", "fields": [
+          { "name": "Partition",
+            "versions": "0+",
+            "type": "int32",
+            "about": "The partition number." },
```

Comment:
The format of the new fields does not match the convention. Please, look at existing fields/fields and update the format accordingly.

Reply:
Thought it was more readable like this, changed it though